scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -83,9 +84,8 @@ emcmake cmake \
 make -j 4
 
 cd ..
-cp build/solc/soljson.js ./
 mkdir -p upload
-cp soljson.js upload/
```

Comment:
Yes, all files in the `upload` directory are uploaded to the "release" page in case this is a release.